x-pack/plugins/task_manager/server/config.test.ts

```diff
@@ -296,4 +284,62 @@ describe('config validation', () => {
       `"[discovery.active_nodes_lookback]: active node lookback duration cannot exceed five minutes"`
     );
   });
+
+  test('should not throw if ephemeral_tasks is defined', () => {
+    const config: Record<string, unknown> = {
+      ephemeral_tasks: {
+        enabled: true,
+        request_capacity: 20,
+      },
+    };
+
+    expect(configSchema.validate(config)).toMatchInlineSnapshot(`
```

Comment:
Probably don't need the snapshot match here, do we?  We just want to make sure it doesn't throw an error?  These snapshots can sometimes be a pain to fix up when unrelated changes happen later.